src/app/profiles/profile-settings-form.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { constants } from './profile-constants';
+import { ProfileSelectors } from './store/profile.selectors';
+import { sortByProp } from '../shared/utils';
+
+@Component({
+  selector: 'app-profile-settings-form',
+  templateUrl: './profile-form.html',
+})
+export class ProfileSettingsFormComponent {
+  cities = sortByProp('name', constants.cities);
```

Comment:
You might want to look at separating the concept of `presentational` vs `container` components. This seems to be doing both, but a good practice is to split the 2 concepts.